src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -200,13 +213,6 @@ public void setSingleRoom(Room target, Room editedRoom) {
     }
     //@@author LeeMingDe
 
-    @Override
-    public Optional<Room> getRoomWithRoomNumber(int roomNumber) {
-        assert (roomNumber > 0) : "Room number should be greater than 0.";
-        return roomList.getRoomWithRoomNumber(roomNumber);
-    }
-
-    //@@author chiamyunqing
```

Comment:
I think this line should be added back cos it's meant for the next function removePatientFromRoom()